handbook/project-management/repositories.md

```diff
@@ -21,57 +25,79 @@ In your README, make sure you include or link to:
 * **Project writeup** - this could be a link to a blog post, a part of your README, or another file that documents your project's journey from start to end. It's a great way to highlight the work that went into your project!
   * For example, the [`ubclaunchpad/food-doods` README](https://github.com/ubclaunchpad/food-doods/blob/master/README.md) documents the team's ideas, pain points they encountered, design mockups, and more.
 
-## Enforcing Standards
+## Code Tooling
+
+Most modern programming languages have things we call "linters" or other static analysis tools to catch "obvious" mistakes and enforce code style and documentation rules. These tools often feature [code editor](/resources/tools.md#writing-code) integrations that can give you feedback on problems and even fix your code as you type!
+
+Code tooling is generally language-specific - for example:
+
+* JavaScript and Typescript projects have [eslint](https://eslint.org/), [Prettier](https://prettier.io/), and more
```

Comment:
nit: include `[tslint](https://palantir.github.io/tslint/)`

Reply:
There's a big deprecation notice on the repository though :/

From what i've seen, the typescript community seems to be moving towards unifying on eslint, which has good support for typescript in my experience